.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '*'
```

Comment:
Should we limit it to only master? Otherwise you'll have 2 builds for every PR you create.

Reply:
Right, I'll limit it to master